types/prosemirror-model/index.d.ts

```diff
@@ -24,6 +24,8 @@ export class ContentMatch {
   findWrappingFor(node: Node): Array<{ type: NodeType, attrs: AnyObject }> | null | undefined;
 }
 export class Fragment {
+  constructor(content: Node[], size: number);
+  size: number;
```

Comment:
This was only [exposed publicly in 0.23](https://github.com/ProseMirror/prosemirror-model/commit/802d7ffb8a34a6ebcaa439bbadf146df7656de1a), so the version of these types should be bumped from 0.21 to 0.23

Reply:
I have now added all changes from version 0.23